nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -1098,20 +1098,23 @@ in
           noCheck = true; # fsck fails on a r/o filesystem
         };
       }
+      (lib.mkIf (config.boot.initrd.systemd.enable && cfg.writableStore) {
```

Comment:
FYI, in the diskless VM PR, I will introduce this definitely and remove any manual mount.

Reply:
What is the diskless VM PR?